payment/domain/payment.go

```diff
@@ -11,10 +9,25 @@ type (
 		Code string
 	}
 
-	// FlowResult contains an url and a type to use to start a flow
+	// FlowResult contains information about a newly started flow
 	FlowResult struct {
-		URL  *url.URL
-		Type string
+		// Status of the payment flow
+		Status string
+		// PlaceOrder indicates if the order should be placed with the beginning of the payment flow
+		PlaceOrder bool
```

Comment:
I would rename PlaceOrder. The name sounds as if it would only be placed if this one is set :)

Reply:
Good point, I renamed to prop to EarlyPlaceOrder to be more speaking.